### PR TITLE
MIAJS-1748 feature/matb: Add 'Level' word translation

### PR DESCRIPTION
**Pull request by matbmapspeople**
I created a property with default value of 'Level'.
When we bind to it inside kiosk we will pipe it through translation for languages that we support with translations.

### CODE REVIEW

File: packages/components/src/components/location-info/location-info.tsx [hunk 1]
```diff
@@ -12,6 +12,11 @@ export class LocationInfo implements ComponentInterface {
      */
     @Prop() location;
 
+    /**
+     * @description Level info.
```
Comment (ammapspeople) [inline packages/components/src/components/location-info/location-info.tsx:16]: "info"? 🤔 Isn't it just a word or a phrasing?

Comment (matbmapspeople) [reply packages/components/src/components/location-info/location-info.tsx]: @ammapspeople fixed ✅

Comment (ammapspeople) [reply packages/components/src/components/location-info/location-info.tsx]: I don't think I would understand what it was about if I was new to using the component. Can you elaborate the description to be more explaining than just "Level"?

Comment (matbmapspeople) [reply packages/components/src/components/location-info/location-info.tsx]: I tried to be more descriptive now!

File: packages/components/src/components.d.ts [hunk 2]
```diff
@@ -1698,6 +1702,10 @@ declare namespace LocalJSX {
     };
     }
     interface MiLocationInfo {
+        /**
+          * @description Level info.
```
Comment (ammapspeople) [inline packages/components/src/components.d.ts:1706]: Run build scrips (or whatever) to make these autogenerated files reflect that is written in the tsx file.

File: packages/components/src/components/location-info/location-info.tsx [hunk 3]
```diff
@@ -12,6 +12,11 @@ export class LocationInfo implements ComponentInterface {
      */
     @Prop() location;
 
+    /**
+     * @description Location's level property that can by translatable for specific languages. By default level value is 'Level'.
```
Comment (ammapspeople) [inline packages/components/src/components/location-info/location-info.tsx:16]: Make the "would grandma understand this" test 😄 
The phrasing "Location's level property" is confusing.
How about

`The word used for "Level" when showing level info. Default is "Level".`

**Comment by matbmapspeople** [comment]
Suggestions applied.

**Comment by ammapspeople** [comment]
Remember to apply a "minor" label to this pull request. That will make sure it is released to NPM as a semantically minor version (it is an addition after all).

**Review by ammapspeople — APPROVED** [review]

**Pull request merged**